runtime/gc_glue_java/ScavengerDelegate.cpp

```diff
@@ -714,6 +733,31 @@ MM_ScavengerDelegate::switchConcurrentForThread(MM_EnvironmentBase *env)
 #endif
     }
 }
+
+void
+MM_ScavengerDelegate::signalThreadsToFlushCaches(MM_EnvironmentBase *envBase)
+{
+	J9InternalVMFunctions const * const vmFuncs = _javaVM->internalVMFunctions;
+	J9VMThread *walkThread = NULL;
+
+	GC_VMThreadListIterator vmThreadListIterator(_javaVM);
+
+	GC_VMInterface::lockVMThreadList(_extensions);
```

Comment:
This can be achieved by passing NULL as the thread to J9SignalAsyncEvent - it will do the locking and signal all threads.

Reply:
I will add more per-thread action than just signal threads in this loop, so I'd rather leave it as is. Again, with some other stub code in this PR, it's waiting for dependent OMR changes, before I can invoke it here.